addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -556,12 +588,20 @@ ListRenderer.include({
                 }
             }
         }
-        var $thead = this.$('thead');
-        $thead.find('th').each((index, th) => {
-            var $th = $(th);
-            $th.css('width', restoreWidths ? this.columnWidths[index] : $th.outerWidth() + 'px');
+        // Freeze table width according to its size in fixed layout
+        const table = this.el.getElementsByTagName('table')[0];
+        const thead = table.getElementsByTagName('thead')[0];
+        table.style.tableLayout = 'fixed';
+        table.style.width = `${table.offsetWidth}px`;
+
+        // Freeze each th width according to their size in auto layout
+        table.style.tableLayout = 'auto';
```

Comment:
isn't it already the case?

Reply:
The table in `fixed` layout adapt its width according to its parent, while it adapts it to its children whil in `auto`. To have the final width, we have to set it to fix, measure it, and then reset it to auto to measure the column widths.